tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -42,26 +43,239 @@ limitations under the License.
 #include "tensorflow/core/util/mkl_util.h"
 
 #ifndef INTEL_MKL_ML
-
 #include "mkldnn.hpp"
 
 using mkldnn::prop_kind;
 using mkldnn::stream;
-
-using mkldnn::convolution_direct;
 using mkldnn::convolution_forward;
+using mkldnn::convolution_direct;
+
 #else
 #include "mkl_dnn.h"
 #include "mkl_dnn_types.h"
 #endif
 
 namespace tensorflow {
 
+#ifndef INTEL_MKL_ML
+
+struct ConvFwdDimensions {
+  memory::dims src_dims;
+  memory::dims filter_dims;
+  memory::dims bias_dims;
+  memory::dims dst_dims;
+  memory::dims strides;
+  memory::dims dilations;
+  memory::dims padding_l;
```

Comment:
In general, please avoid single letter abbreviations in variable names. So use padding_left, src_dims etc. I'm not even sure I know that "pl" and "pr" stand for below (padding_left and padding_right?).  I'm OK with "dims", "src", "dst" since they are pretty standard in the code base.

Same everywhere.

Reply:
Based on suggested,  all non-std names (padding_l/_l, pl and pr) has been renamed with explicit meanings (padding_left/_right). Thanks!